src/MDOP.jl

```diff
@@ -87,3 +220,134 @@ function beta_statistic(Y::Dataset, s::Vector; τs = 0:50 , w::Int = 1)
     β = mean(log10.(Δϕ); dims=1)
     return vec(β)
 end
+
+
+"""
+    embed2(Y::Dataset,s::Vector,τ::Int) -> Y_new
+adds the `τ` lagged values of the time series `s` as additional component to the
+embedding vector `Y`.
+"""
+function embed2(Y::Dataset,s::Vector{Float64},τ::Int)
```

Comment:
Please consider adding a white space after each comma: `Y, s, τ` instead of `Y,s,τ`, because it matches in style with the rest of the source code and thus makes it more readable :) 

(also, same comment as above for the assertion `s::Vector{Float64} -> s::Vector{<:Real}`.